plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -55,6 +55,10 @@ class SentryPlugin : Plugin<Project> {
                 extraProperties.get(SENTRY_PROJECT_PARAMETER).toString()
             }.getOrNull()
 
+            // temp folder for sentry-related stuff
```

Comment:
Should this be created with `Files.createTempDir()` instead?

Reply:
hm, we want this specifically to be placed into `build/tmp/sentry` folder of the target project, so the users can access it if necessary - not sure if `createTempDir` is suitable for that

Reply:
if the idea is that they collect those files in case there's a problem and we have to investigate it, `build/tmp/sentry` makes it easier